src/entities/Blog.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  PrimaryGeneratedColumn,
+} from 'typeorm'
+
+@Entity()
+export class Blog {
+  @PrimaryGeneratedColumn('uuid')
+  id!: string
+
+  @Column()
+  author_id!: string
```

Comment:
Here you have to connect author and blog with foreign key. It is good for security and retrieving data from database. User and its articles and article with its user. Hint: use @OneToMany, @ManyToOne decorators

Reply:
when i create User entity it did not have blogs field. To correct my mistake on Blog entity I added blogs filed on User entity.